tools/processresults.py

```diff
@@ -8,11 +8,13 @@
 import dateutil.parser
 from datetime import datetime
 import json
+import logging
 import re
 
 DOGFOOD_START_DATE = datetime(2014, 12, 01, 0, 0, 0, 0)
 DEMOGRAPHIC_STABLE_DATE = datetime(2014, 12, 18, 0, 0, 0, 0)
 FRIENDS_AND_FAMILY_BETA_DATE = datetime(2015, 4, 9, 0, 0, 0, 0)
+PUBLIC_LAUNCH_DATE = datetime(2015, 5, 11, 13, 30, 0, 0)
```

Comment:
We don't use the timezone in code (since survey responses are recorded in UTC, and this time is also UTC), but I've added a comment to clarify that this is specified in UTC.